demo/routes/enrolment.js

```diff
@@ -1,80 +1,125 @@
-const config = require('../config')
+// const config = require('../config')
+const serviceLookup = require('../lib/services')
+
+/**
+ * filter the available services to only those for which there is an enrolment request
+ * @param {Object} list
+ * @param {Array<EnrolmentRequest} requests
+ */
+const filterServiceLookup = (list, requests) => {
+  const allowedServices = requests.map(r => r.serviceId)
+  const result = {}
+  Object.keys(list).forEach((key) => {
```

Comment:
I think wherever possible we should use native code rather than utility libraries. I've weaned myself off lodash see here: `https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore`